src/3_PB/AnalisiDeiRequisiti/content.typ

```diff
@@ -648,21 +673,20 @@ Relativamente all'utilizzo della #glossary("dashboard"), viene definito un unico
   + il #glossary("sistema") aggiorna tutti i #glossary("pannelli") collegati.
 */
 /*Cambiare il numero nell'immagine*/
+#pagebreak()
 === UC7: Ordinamento #glossary("pannelli") tabellari
 - *Attore Principale*: #glossary("amministratore pubblico")\;
 - *Precondizioni*: l'#glossary("amministratore pubblico") sta visualizzando un #glossary("pannello"), con all'interno una tabella; 
 - *Postcondizioni*: l'#glossary("amministratore pubblico") visualizza i dati ordinati nella tabella, secondo tale campo;
 - *Scenario Principale*:
-  + L'#glossary("amministratore pubblico") seleziona un campo della tabella, secondo cui ordinare i dati, tra i seguenti: nome, percentuale di batteria, data di manutenzione del #glossary("sensore"), valore della misurazione e timestamp della rilevazione (ove presenti);
+  + L'#glossary("amministratore pubblico") seleziona un campo della tabella, secondo cui ordinare i dati, tra i seguenti: nome del #glossary("sensore"), tipologia del #glossary("sensore"), valore della misurazione inviata e timestamp della rilevazione (ove presenti);
   + Per tale campo l'#glossary("amministratore pubblico") sceglie tra l'ordinamento crescente e decrescente.
```

Comment:
qui ho un dubbio
l'ordinamento crescente e decrescente non sono 2 funzionalità diverse secondo cui ordinare i dati e per tale motivo messe come use case separati?

Reply:
mi stai dicendo che faresti UC7 ordinamento con specializzazioni UC7.1 e UC7.2 per crescente e decrescente? 
ci può stare ma sai cosa stavo pensando io atm? che forse non è giusto schiacciare l'ordinamento di tutti i campi all'interno dello stesso UC e sarebbe meglio fare dei sotto-UC (7.1, 7.2..), uno per campo? e poi ognuno di questi avrebbe 2 specializzazioni crescente e decrescente

@Dovid308 cosa ne pensi?

Reply:
non credo serva spaccare

Reply:
per il momento non ho spaccato ma dato che per il filtraggio in UC6 e avanti abbiamo fatto una cosa simile, considererei l'idea di farlo anche per l'ordinamento per ciascun campo; cmq ne riparliamo prossimamente